README.md

```diff
@@ -0,0 +1,16 @@
+### How help-activity works:
+
```

Comment:
Help Activity is also a activity itself, where yo u can get all the help content, Would be good have a paragraph about that.

Reply:
added.